PreFreeSurfer/PreFreeSurferPipeline.sh

```diff
@@ -355,6 +377,33 @@ RUN=`opts_GetOpt1 "--printcom" $@`
 UseJacobian="$(echo ${UseJacobian} | tr '[:upper:]' '[:lower:]')"
 UseJacobian=`opts_DefaultOpt $UseJacobian "true"`
 
+
+# ------------------------------------------------------------------------------
+#  Compliance check
+# ------------------------------------------------------------------------------
+
+MPPMode=`opts_GetOpt1 "--mpp-mode" $@`
+MPPMode=`opts_DefaultOpt $MPPMode "HCPStyleData"`
```

Comment:
Move the arg parsing and default setting up into those respective sections of code (same as with PostFreeSurferPipeline).

Reply:
Ok.